src/refactor/cli.py

```diff
@@ -0,0 +1,37 @@
+'''
+ * Copyright (C) 2022  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+'''
+'''
+ * @file cli.py
+ * @author Robert Taylor
+ * @date 2022-09-21
+'''
+
+from groundStation import GroundStation
+from options import optionsFactory
+
+class cli(GroundStation):
```

Comment:
Do we usually capitalize class names?

Reply:
Yes, though I'm not sure how to capitalize an acronym. @JenishPatel99 Thoughts?